attributes/default.rb

```diff
@@ -88,9 +93,13 @@
 # Set prefix to '' if you want Chef tags to be sent without prefix.
 default['datadog']['tag_prefix'] = 'tag:'
 
-# Don't change these
-# The host of the Datadog intake server to send agent data to
-default['datadog']['url'] = 'https://app.datadoghq.com'
+# Don't change these.
+# The host of the Datadog intake server to send Agent data to, only set this option
+# if you need the Agent to send data to a custom URL.
+# The nil value will let the Agent 6 select the URL to send the data.
```

Comment:
Let's add that `Any non-nil value overrides the 'site' value, prefer using 'site' unless your use case isn't covered by 'site'.` (might be a bit long, but I want to encourage users to use `site` instead of `url`, unless they have good reasons to use `url` )